Geometry_Engine/Query/IsEqual.cs

```diff
@@ -117,14 +117,6 @@ public static bool IsEqual(this Line line, Line other, double tolerance = Tolera
 
         /***************************************************/
 
-        [NotImplemented]
-        public static bool IsEqual(this NurbsCurve curve, NurbsCurve other, double tolerance = Tolerance.Distance)
```

Comment:
anopther one I think we just should implement rather than remove, but might be out of scope for this PR

Reply:
Sure. I think I can implement that one together with #1955

Reply:
Given another thought i decided to raise another issue for that (#1958) and resolve it after merging to keep code more consistent about not making `[NotImplemented]` methods a "todo list".